pkg/ipam/ipam.go

```diff
@@ -155,7 +165,11 @@ func (i *IPAM) useFinalizerIfNeeded(eip *networkv1alpha1.Eip) (bool, error) {
 }
 
 func (i *IPAM) AssignIP(serv *corev1.Service) (*AssignIPResponse, error) {
-	return i.ds.AssignIP(serv.Name, serv.Namespace)
+	lbType := serv.Annotations[constant.PorterLBTypeAnnotationKey]
+	if lbType == "" {
+		return nil, errors.NewEIPLBTypeNotFoundError()
```

Comment:
default type should be bgp

Reply:
I don't think we need to make decisions for users